libs/api/domains/transport-authority/transfer-of-vehicle-ownership/src/lib/graphql/main.resolver.ts

```diff
@@ -0,0 +1,20 @@
+import { Query, Resolver } from '@nestjs/graphql'
+import { ApiScope } from '@island.is/auth/scopes'
+import { UseGuards } from '@nestjs/common'
+import { IdsUserGuard, Scopes, ScopesGuard } from '@island.is/auth-nest-tools'
+import { InsuranceCompany } from './models'
+import { TransferOfVehicleOwnershipApi } from '../transferOfVehicleOwnership.service'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Scopes(ApiScope.internal)
+@Resolver()
```

Comment:
We might want to change this to be a "public" resolver (no scope check) so it works for different delegations later on.